src/Core/TypeReaders/GuildUserProxyTypeReader.cs

```diff
@@ -10,6 +10,9 @@ namespace BrackeysBot
 {
     public class GuildUserProxyTypeReader : UserTypeReader<IGuildUser>
     {
+        // Found this value on https://www.pixelatomy.com/snow-stamp/ -> Enter snowflake of 1, copy UNIX value.
+        private readonly int DISCORD_START_UNIX = 1420070400;
```

Comment:
Could've also found in the API reference: https://discord.com/developers/docs/reference#snowflakes-snowflake-id-format-structure-left-to-right.
😜

Reply:
TIL